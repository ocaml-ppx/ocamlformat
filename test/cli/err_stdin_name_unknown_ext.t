  $ ocamlformat --name a.cpp -
  ocamlformat: Cannot deduce file kind from passed --name. Please specify --impl or --intf
  [1]
