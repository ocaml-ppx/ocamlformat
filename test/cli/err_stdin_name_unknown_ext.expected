ocamlformat: Cannot deduce file kind from passed --name. Please specify --impl or --intf
